src/pages/settings/EditPasswordScreen.js

```diff
@@ -0,0 +1,149 @@
+            errors.newPassword = this.props.translate(this.errorKeysMap.newPassword);
+        }
+
+        if (values.currentPassword && values.newPassword && _.isEqual(values.currentPassword, values.newPassword)) {
+            errors.newPasswordSameAsOld = this.props.translate(this.errorKeysMap.newPasswordSameAsOld);
+        }
+        return errors;
+    }
+
+    /**
+     * @param {Object} values - form input values passed by the Form component
+     */
+    submit(values) {
+        User.updatePassword(values.currentPassword, values.newPassword);
+    }
+
+    render() {
+        return (
+            <ScrollView
+                style={styles.flex1}
+                contentContainerStyle={styles.p5}
+
+                // Allow the user to click show password while password input is focused.
+                // eslint-disable-next-line react/jsx-props-no-multi-spaces
+                keyboardShouldPersistTaps="always"
+            >
+                <Text style={[styles.mb6]}>
+                    {this.props.translate('passwordPage.changingYourPasswordPrompt')}
+                </Text>
+                <Form
+                    formID={ONYXKEYS.FORMS.EDIT_PASSWORD_FORM}
+                    validate={this.validate}
+                    onSubmit={this.submit}
+                    submitButtonText={this.props.translate('common.save')}
+                >
+                    <View style={styles.mb6}>
+                        <TextInput
+                            label={`${this.props.translate('passwordPage.currentPassword')}*`}
+                            secureTextEntry
+                            autoCompleteType="password"
+                            textContentType="password"
+                            returnKeyType="done"
+                            inputID="currentPassword"
+                        />
+                    </View>
+                    <View style={styles.mb6}>
+                        <TextInput
+                            label={`${this.props.translate('passwordPage.newPassword')}*`}
+                            secureTextEntry
+                            autoCompleteType="password"
+                            textContentType="password"
+                            inputID="newPassword"
+                        />
+
+                        <Text
+                            style={[
+                                styles.textLabelSupporting,
+                                styles.mt1,
+                            ]}
+                        >
+                            {this.props.translate('passwordPage.newPasswordPrompt')}
+                        </Text>
+
+                    </View>
+                    { !_.isEmpty(this.props.account.errors) && (
+                        <Text style={styles.formError}>
+                            {ErrorUtils.getLatestErrorMessage(this.props.account)}
+                        </Text>
+                    )}
```

Comment:
We shouldn't need this. Form will automatically display the errors passed to the FormID key. I'm not sure that we need to subscribe to the account Onyx key.

Reply:
<img width="398" alt="Screen Shot 2022-11-04 at 15 43 12" src="https://user-images.githubusercontent.com/113963320/199930125-06e35552-29d3-40ec-a535-68cd3891f45b.png">

As I commented here https://github.com/Expensify/App/pull/12032#discussion_r1013727840, I think we still need to subscribe to the account to get the error.

Reply:
Yea, that's ok. Ideally the API would return the error to the `EDIT_PASSWORD_FORM` key, but I'm not sure if we need to refactor that right now so we can keep it.